src/locales/de.json

```diff
@@ -176,7 +176,6 @@
       "memory_usage": "Usage - {{value}} GB-Hours"
     },
     "change_column_title": "Month Over Month Change",
-    "charge_column_subtitle": " (Total {{total}})",
     "charge_column_title": "Charges",
```

Comment:
This line was changed in `en.json` but not here.

Reply:
Thanks. The linter didn't catch that.